.github/workflows/ci_suite.yml

```diff
@@ -90,9 +90,6 @@ jobs:
           mysql -u root -proot -e 'CREATE DATABASE tg_ci_prefixed;'
       - name: Install rust-g
         run: |
-          sudo dpkg --add-architecture i386
-          sudo apt update || true
-          sudo apt install -o APT::Immediate-Configure=false libssl1.1:i386
           bash tools/ci/install_rust_g.sh
```

Comment:
Do you need to readd?

Reply:
No, I do that in the install_rust instead